README.md

```diff
@@ -149,7 +143,7 @@ https://download.swift.org/repo/centos/releases/8/swiftlang.repo
 	* [GitHub discussion](https://github.com/apple/swift-installer-scripts/pull/37#discussion_r726707320)
 * Should we support multiple swiftlang versions on the system?
 * Multiple packages:
-	* swiftlang
+	* [swiftlang](https://www.swiftlang.xyz)
```

Comment:
I don't follow why we are linking this in the README.

Reply:
Thanks for reviewing my request. 
Yeah that’s right we don’t need to merge line 146 only